### PR TITLE
Show virtual participations on contest scoreboard

### PR DESCRIPTION
**Pull request by ngkan**

### CODE REVIEW

**Comment by leduythuccs** [comment]
After change:
image

**Comment by ngkan** [comment]
Latest preview

image

File: templates/contest/ranking-table.html [hunk 1]
```diff
@@ -20,6 +20,19 @@
     </td>
 {% endblock %}
 
+<!-- How to apply ordinal in jinja2 ??? -->
```
Comment (leduythuccs) [inline templates/contest/ranking-table.html:23]: maybe we can create a new filter.

**Review by leduythuccs — APPROVED** [review]
LGTM

**Pull request merged**